python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -369,6 +370,22 @@ class IsrTaskLSSTConfig(pipeBase.PipelineTaskConfig,
         default=False,
     )
 
+    # Amp offset correction.
+    doMeasureAmpOffset = pexConfig.Field(
```

Comment:
As above, `doAmpOffset` would control whether the exposure is passed to `AmpOffsetTask`, and that task's `doApplyAmpOffset` would control if the measured values are subtracted or not.